src/Chapter2.hs

```diff
@@ -621,7 +623,9 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven (x:y:zs) = x : takeEven zs
+takeEven list = if length list == 1 then list else []
```

Comment:
Actually the both cases return list itself, so no need to check 🙂 
Alternatively, you can use pattern matching to check that, it is better than using `length` generally 🚤